### PR TITLE
Created new css files and changed existing ones, created buttons for click handler and mapping out style selector

### PR DESCRIPTION
**Pull request by quyencodes**
- Created separate css files in `productOverview`
- Created button elements in `ProductInformation.jsx`
- Created `StyleSelector` component
- Changes to `searchBar` css
- Front end looks closer to sample provided

### CODE REVIEW

File: src/components/ProductOverview/index.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { useContext, useEffect, useState } from 'react'
 import ImageGallery from './PO-Components/ImageGallery'
 import Description from './PO-Components/Description'
 import ProductInformation from './PO-Components/ProductInformation'
-import '@/styles/productOverview.css'
+import '@/styles/productOverview/productOverview.css'
```
Comment (lgoodcode) [inline src/components/ProductOverview/index.jsx:6]: Maybe `main.css` instead of repeating the container name?

File: src/styles/globals.css [hunk 2]
```diff
@@ -11,7 +11,7 @@
   font-style: normal;
   font-weight: 300 800;
   font-display: swap;
-  src: url(../assets/fonts/Inter.woff2) format('woff2');
+  src: url(@/assets/fonts/Inter.woff2) format('woff2');
```
Comment (lgoodcode) [inline src/styles/globals.css:14]: Noice use of the alias. I'm gonna do that, didn't even think of that.

File: src/components/ProductOverview/index.jsx (hunk 1 above)
Comment (quyencodes) [reply src/components/ProductOverview/index.jsx:6]: forsure say less

**Review by lgoodcode — APPROVED** [review]
Noice. Check comments.

File: src/styles/globals.css (hunk 2 above)
Comment (quyencodes) [reply src/styles/globals.css:14]: all you king thanks for reviewing

**Pull request merged**